### PR TITLE
Standardize comments and separator lines in serialize, super3d, and tests directories.

### PR DESCRIPTION
**Pull request by gbrooksby**

### CODE REVIEW

File: arrows/serialize/json/load_save.cxx [hunk 1]
```diff
@@ -221,38 +221,36 @@ void load( ::cereal::JSONInputArchive& archive, ::kwiver::vital::detected_object
...
- *
- * You can argue that endianess is a message property, but we can't
- * access message level attributes way down here in the serializer.
- *
- * Note that this is only a problem with images where the pixels are
- * multi-byte.
- */
-
-// ============================================================================
+//  We still may have a problem in this serialization approach if the
```
[the remark this answers is not in the document]
Comment (gbrooksby) [reply arrows/serialize/json/load_save.cxx:224]: I can't spot the added space you're referring to.  I have fixed the comments to use 3 slashes rather then 2 though.

File: arrows/serialize/json/tests/test_serialize.cxx [hunk 2]
```diff
@@ -238,12 +236,11 @@ TEST( serialize, bounding_box )
...
-  std::cout << "bbox_dser { " << bbox_dser.min_x() << ", "
-            << bbox_dser.min_y() << ", "
-            << bbox_dser.max_x() << ", "
-            << bbox_dser.max_y() << "}\n";
-  */
+  // useful for debugging
+  // std::cout << "bbox_dser { " << bbox_dser.min_x() << ", "
+  //         << bbox_dser.min_y() << ", "
+  //         << bbox_dser.max_x() << ", "
+  //         << bbox_dser.max_y() << "}\n";
```
[the remark this answers is not in the document]
Comment (gbrooksby) [reply arrows/serialize/json/tests/test_serialize.cxx:243]: I've reverted this to the previous form.

File: arrows/serialize/json/load_save_point.h [hunk 3]
```diff
@@ -13,7 +13,7 @@ namespace cereal {
   class JSONOutputArchive;
   class JSONInputArchive;
 
-// ---- points
+// --- points
```
[the remark this answers is not in the document]
Comment (gbrooksby) [reply arrows/serialize/json/load_save_point.h:16]: I agree.  I've reverted this to the previous form.  It's at least consistent with the other comment in the file like it.

File: arrows/serialize/protobuf/convert_protobuf.h [hunk 4]
```diff
@@ -57,7 +57,7 @@ namespace arrows {
 namespace serialize {
 namespace protobuf {
 
-// ---- activity
+// ----  activity
```
[the remark this answers is not in the document]
Comment (gbrooksby) [reply arrows/serialize/protobuf/convert_protobuf.h:60]: Agreed. I've fixed both instances.

File: arrows/serialize/protobuf/tests/test_serialize.cxx [hunk 5]
```diff
@@ -202,12 +200,10 @@ TEST( serialize, bounding_box )
...
-  /*
-  std::cout << "bbox_dser { " << bbox_dser.min_x() << ", "
-            << bbox_dser.min_y() << ", "
-            << bbox_dser.max_x() << ", "
-            << bbox_dser.max_y() << "}\n";
-  */
+  // std::cout << "bbox_dser { " << bbox_dser.min_x() << ", "
+  //         << bbox_dser.min_y() << ", "
+  //         << bbox_dser.max_x() << ", "
+  //         << bbox_dser.max_y() << "}\n";
```
[the remark this answers is not in the document]
Comment (gbrooksby) [reply arrows/serialize/protobuf/tests/test_serialize.cxx:206]: I've reverted this one.

File: arrows/serialize/json/load_save.cxx (hunk 1 above)
[the remark this answers is not in the document]
Comment (mwoehlke-kitware) [reply arrows/serialize/json/load_save.cxx]: Originally, there was one space between the leading `*` and text. Now there are two spaces after the `//`, which has caused everything to shift over one column.

[the remark this answers is not in the document]
Comment (mwoehlke-kitware) [reply arrows/serialize/json/load_save.cxx]: Also, I don't believe this is intended to be a Doxygen (`///`) comment. (Which caused everything to shift over *two* columns due to the added third `/`...)

[the remark this answers is not in the document]
Comment (gbrooksby) [reply arrows/serialize/json/load_save.cxx]: I changed it back to `//` now with the extra space removed.

**Pull request merged**